core-web/libs/edit-content/src/lib/fields/dot-edit-content-binary-field/components/dot-binary-field-editor/dot-binary-field-editor.component.spec.ts

```diff
@@ -126,17 +127,25 @@ describe('DotBinaryFieldEditorComponent', () => {
 
     describe('Editor', () => {
         it('should set editor language', fakeAsync(() => {
+            const expectedMonacoOptions = {
+                ...DEFAULT_BINARY_FIELD_MONACO_CONFIG,
+                language: 'javascript'
+            };
+
+            spectator.detectChanges();
+
             component.form.setValue({
                 name: 'script.js',
                 content: 'test'
             });
 
-            tick(1000);
+            component.form.updateValueAndValidity();
```

Comment:
Doesn't this get's trigger automatically when the `setValue` get's called?

Reply:
Yes. It does. Removed, Only is necessary if we manipulate the validators.